src/esm_runscripts/oasis.py

```diff
@@ -98,12 +100,12 @@ def add_coupling(
             sep = ":"
 
         if export_mode == "DEFAULT":
-            if lresume == False:
+            if lresume is False:
```

Comment:
Please, revert this change in the new PR. As I said `0 == False` gives you `True`. `0 is False` gives you `False`